### PR TITLE
Update dev.md

### PR DESCRIPTION
**Pull request by ruaridhg**
Added dev tools for environments, refactoring, logging and monitoring

### CODE REVIEW

**Comment by ruaridhg** [comment]
https://github.com/ml-tooling/best-of-python-dev

File: pages/dev.md [hunk 1]
```diff
@@ -4,3 +4,12 @@ layout: default
 ---
 
 # Development tools
```
Comment (alessandrofelder) [inline pages/dev.md:6]:
```suggestion
# Integrated Development Environments (IDEs)
 Name     | Short description | 🚥 |
| -------- | ------------------| ---------------|
| [Visual Studio Code](https://code.visualstudio.com/docs) | A light-weight general-purpose IDE |  |
| [PyCharm](https://www.jetbrains.com/pycharm/) | A Python IDE by JetBrains |  |

# Development tools
```

**Review by alessandrofelder — CHANGES_REQUESTED** [review]
IDEs need adding

**Review by alessandrofelder — APPROVED** [review]
👍

**Pull request merged**